notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4410,6 +4410,56 @@ the desire for vengeance [A soldier tried to kill Deunan to avenge her father ha
 mass infertility [Although not really explored in any way, Seven Elders revealed that they planned to release a virus that would make the human race permanently unable to reproduce.]
 
 
+movie: Casshern (2004)
+======================
+
+:: Title
+Casshern
+
+:: Description
+Casshern (キャシャーン, Kyashān) is a 2004 Japanese tokusatsu superhero film
+adaptation of the anime series of the same name. It was written and directed
+by Kazuaki Kiriya.
+
+Synopsis: A race of artificially made humans wage war on mankind.
+
+:: Date
+2004-04-24
+
+:: References
+https://en.wikipedia.org/wiki/Casshern_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+war [The Neo Sapiens waged war on the Eastern Federation. The Eastern Federation mobilized its military against a rebel region. The story is set in the immediate aftermath of a fifty-year war between two rival world powers: the Eastern Federation and Europa.]
```

Comment:
Sounds like there ought to be an existential risk here as human civilisation would be wiped out by something it had created - though tough one as it would presumably be replaced by a neo human civilisation.

Reply:
Hum... We have **rise of the machines** (not quite) and **displacement by a rising new species** (also not quite). Use the general **anthropogenic existential risk to civilization** theme, and consider adding something about "humanity being destroyed by something of its own creation" later?